client/my-sites/upgrades/checkout-thank-you/jetpack-thank-you-card.jsx

```diff
@@ -0,0 +1,359 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import page from 'page';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import filter from 'lodash/filter';
+import range from 'lodash/range';
+import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import PlanThankYouCard from 'blocks/plan-thank-you-card';
+import FeatureExample from 'components/feature-example';
+import Notice from 'components/notice';
+import NoticeAction from 'components/notice/notice-action';
+import Spinner from 'components/spinner';
+import Gridicon from 'gridicons';
+import QueryPluginKeys from 'components/data/query-plugin-keys';
+import analytics from 'lib/analytics';
+import JetpackSite from 'lib/site/jetpack';
+import support from 'lib/url/support';
+
+// Redux actions & selectors
+import { getSelectedSite, getSelectedSiteId } from 'state/ui/selectors';
+import { isJetpackSite, isRequestingSites, getRawSite } from 'state/sites/selectors';
+import { getPlugin } from 'state/plugins/wporg/selectors';
+import { fetchPluginData } from 'state/plugins/wporg/actions';
+import { requestSites } from 'state/sites/actions';
+import {
+	installPlugin,
+} from 'state/plugins/premium/actions';
+import {
+	getPluginsForSite,
+	getActivePlugin,
+	getNextPlugin,
+	isFinished,
+	isInstalling,
+	isRequesting,
+	hasRequested
+} from 'state/plugins/premium/selectors';
+// Store for existing plugins
+import PluginsStore from 'lib/plugins/store';
+
+class JetpackThankYouCard extends Component {
+	trackConfigFinished( eventName, options = null ) {
+		if ( ! this.sentTracks ) {
+			analytics.tracks.recordEvent( eventName, options );
+		}
+		this.sentTracks = true;
+	}
+
+	trackManualInstall() {
+		analytics.tracks.recordEvent( 'calypso_plans_autoconfig_click_manual_error' );
+	}
+
+	trackManagePlans() {
+		analytics.tracks.recordEvent( 'calypso_plans_autoconfig_click_manage_plans' );
+	}
+
+	trackContactSupport() {
+		analytics.tracks.recordEvent( 'calypso_plans_autoconfig_click_contact_support' );
+	}
+
+	// plugins for Jetpack sites require additional data from the wporg-data store
+	addWporgDataToPlugins( plugins ) {
+		return plugins.map( plugin => {
+			const pluginData = getPlugin( this.props.wporg, plugin.slug );
+			if ( ! pluginData ) {
+				this.props.fetchPluginData( plugin.slug );
+			}
+			return Object.assign( {}, plugin, pluginData );
+		} );
+	}
+
+	allPluginsHaveWporgData() {
+		const plugins = this.addWporgDataToPlugins( this.props.plugins );
```

Comment:
We may not need the wporg plugins data any longer since we're currently just using the plugin slug. I believe that's a holdover from the old UI where we wanted to show the plugin icon and the plugin name. I'll look into that in future iterations.